qase-pytest/changelog.md

```diff
@@ -0,0 +1,41 @@
+# qase-pytest 6.1.0b1
+
+## What's new
+
+Implement a method which constructs the signature to each test result.
+It takes the file path, the suites, the qase ids and the parameters.
+
+# qase-pytest 6.0.3
+
+## What's new
+
+Fixed the issue then specified suite in the decorator was not displayed. Fix [#231]
+
+# qase-pytest 6.0.2
+
+## What's new
+
+Fixed the issue [#226]:
+
+```log
+TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
+```
+
+# qase-pytest 6.0.1
+
+## What's new
+
+Change a name of the complete test run parameter for CLI arguments. Fix [#160]
```

Comment:
```suggestion
Changed the name of the complete test run parameter for CLI arguments. Fix [#160]
```

Reply:
what's the old and the new name?

Reply:
old name - qase_testops_run_description
new name - qase_testops_run_complete

but it was in the internal logic. For clients, the names have not changed.